README.md

```diff
@@ -14,7 +14,7 @@ This project is currently under development and can have breaking API changes.
 
 ## Requirements
 
-- Swift 5.4+
+- Swift 5.6+
```

Comment:
```suggestion
- Swift 5.6
```

No + anymore 😆

Reply:
fair fair. we just gotta remember to update this once 6 comes out.